.travis.yml

```diff
@@ -72,10 +54,49 @@ install:
     # install yt
     pip install -e .
 
-script:
-  - |
-    coverage run $(which nosetests) -c nose.cfg yt
-    coverage report
+jobs:
+  include:
+    - stage: lint
+      python: 3.4
+      env: NUMPY=numpy==1.10.4 CYTHON=cython==0.24 MATPLOTLIB= SYMPY= H5PY= SCIPY= FASTCACHE= IPYTHON=
+      script: flake8 yt/
+
+    - stage: tests
+      python: 2.7
+      env: NUMPY=numpy==1.10.4 CYTHON=cython==0.24 MATPLOTLIB=matplotlib==1.5.3 SYMPY=sympy==1.0 H5PY= SCIPY= FASTCACHE= FLAKE8= IPYTHON=ipython==1.0
+      script: coverage run $(which nosetests) -c nose.cfg yt
+
+    - stage: tests
+      python: 2.7
+      env: FLAKE8=
+      script: coverage run $(which nosetests) -c nose.cfg yt
```

Comment:
what's the difference between this stage and the previous one?

Reply:
The previous one tests the minimum versions of our dependencies. This one tests python2.7 with the latest versions from pip.